### ISSUE TITLE
enable_vpn broked on static IPs

### ISSUE THREAD

**Issue by sebbbastien** [gabstopper/ansible-stonesoft#34]
Hi David,

I've a problem with `enable_vpn`. Enabling enpoint on DHCP interfaces is ok, but interfaces with static IP is not.

This file creates a 110 template with DHCP interfaces:
```yaml
- name: Firewall Template
  hosts: localhost
  gather_facts: no

  tasks:
    - name: Layer 3 FW template
      engine:
        primary_mgt: 0
        domain_server_address:
          - name: 8.8.8.8
            type: ipaddress
        file_reputation: false

        name: FW110
        backup_mgt: SWP_0.1
        auth_request: SWP_0.8

        type: single_fw

        interfaces:
          - interface_id: 0
            interfaces:
              - nodes:
                  - dynamic: true
                    dynamic_index: 1
            comment: "1st ISP"
            zone_ref: External
          - interface_id: 1
            interfaces:
              - nodes:
                  - address: 1.1.1.1
                    network_value: 1.1.1.0/24
                    nodeid: 1
                vlan_id: 10
                zone_ref: Internal
              - nodes:
                  - address: 1.1.2.1
                    network_value: 1.1.2.0/24
                    nodeid: 1
                vlan_id: 20
                zone_ref: Internal
          - appliance_switch_module: 110
            interface_id: SWP_0
            type: switch_physical_interface
            port_group_interface:
              - interface_id: SWP_0.1
                interfaces:
                  - nodes:
                      - dynamic: true
                        dynamic_index: 2
                switch_physical_interface_port:
                  - switch_physical_interface_port_comment: ""
                    switch_physical_interface_port_number: 1
                zone_ref: External
              - interface_id: SWP_0.2
                interfaces:
                  - nodes:
                      - dynamic: true
                        dynamic_index: 3
                switch_physical_interface_port:
                  - switch_physical_interface_port_comment: ""
                    switch_physical_interface_port_number: 2
                zone_ref: External
              - interface_id: SWP_0.3
                interfaces:
                  - nodes: []
                switch_physical_interface_port:
                  - switch_physical_interface_port_comment: ""
                    switch_physical_interface_port_number: 3
                zone_ref: External
              - interface_id: SWP_0.8
                interfaces:
                  - nodes:
                      - address: 12.12.12.12
                        network_value: 12.12.12.0/24
                        nodeid: 1
                        auth_request: True
                switch_physical_interface_port:
                  - switch_physical_interface_port_comment: "LAN"
                    switch_physical_interface_port_number: 7
                zone_ref: Internal
        enable_vpn:
          - "First DHCP Interface ip"
          - "Second DHCP Interface ip"
          - "DHCP Interface 3 ip"
        delete_undefined_interfaces: True

        smc_logging:
          level: 10
          path: ansible-smc.log
```
This file fail on `enable_vpn`:
```yaml
- name: Firewall Template
  hosts: localhost
  gather_facts: no

  tasks:
    - name: Layer 3 FW template
      engine:
        primary_mgt: 0
        domain_server_address:
          - name: 8.8.8.8
            type: ipaddress
        file_reputation: false

        name: FW330_cluster
        backup_mgt: "1.10"
        auth_request: 0
        enable_vpn:
          - "1.1.1.1"

        type: fw_cluster
        cluster_mode: balancing

        interfaces:
          - interface_id: 0
            cvi_mode: packetdispatch
            macaddress: 02:02:02:20:20:11
            interfaces:
              - cluster_virtual: 1.1.1.1
                network_value: 1.1.1.0/24
                nodes:
                  - address: 1.1.1.2
                    network_value: 1.1.1.0/24
                    nodeid: 1
                  - address: 1.1.1.3
                    network_value: 1.1.1.0/24
                    nodeid: 2
            comment: "1st ISP"
            zone_ref: External
          - interface_id: 1
            cvi_mode: packetdispatch
            macaddress: 02:02:02:20:20:22
            interfaces:
              - cluster_virtual: 1.1.2.1
                network_value: 1.1.2.0/24
                vlan_id: 10
                nodes:
                  - address: 1.1.2.2
                    network_value: 1.1.2.0/24
                    nodeid: 1
                  - address: 1.1.2.3
                    network_value: 1.1.2.0/24
                    nodeid: 2
            comment: "2nd ISP"
            zone_ref: External
          - interface_id: 8
            cvi_mode: packetdispatch
            macaddress: 02:02:02:20:20:33
            interfaces:
              - cluster_virtual: 1.1.3.1
                network_value: 1.1.3.0/24
                nodes:
                  - address: 1.1.3.2
                    network_value: 1.1.3.0/24
                    nodeid: 1
                  - address: 1.1.3.3
                    network_value: 1.1.3.0/24
                    nodeid: 2
            comment: "3nd ISP"
            zone_ref: External

        delete_undefined_interfaces: True

        smc_logging:
          level: 10
          path: ansible-smc.log
```

Partial Ansible output:
```
Using module file /home/seb/tmp/temp/ansible-stonesoft/library/engine.py
<127.0.0.1> ESTABLISH LOCAL CONNECTION FOR USER: seb
<127.0.0.1> EXEC /bin/sh -c '/home/seb/tmp/temp/env/bin/python3 && sleep 0'
fatal: [localhost]: FAILED! => {
    "changed": false,
    "module_stderr": "<stdin>:17: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses\nTraceback (most recent call last):\n  File \"<stdin>\", line 113, in <module>\n  File \"<stdin>\", line 105, in _ansiballz_main\n  File \"<stdin>\", line 48, in invoke_module\n  File \"/usr/lib/python3.7/imp.py\", line 234, in load_module\n    return load_source(name, filename, file)\n  File \"/usr/lib/python3.7/imp.py\", line 169, in load_source\n    module = _exec(spec, sys.modules[name])\n  File \"<frozen importlib._bootstrap>\", line 630, in _exec\n  File \"<frozen importlib._bootstrap_external>\", line 728, in exec_module\n  File \"<frozen importlib._bootstrap>\", line 219, in _call_with_frames_removed\n  File \"/tmp/ansible_engine_payload_7iohzesx/__main__.py\", line 1742, in <module>\n  File \"/tmp/ansible_engine_payload_7iohzesx/__main__.py\", line 1739, in main\n  File \"/tmp/ansible_engine_payload_7iohzesx/__main__.py\", line 834, in __init__\n  File \"/tmp/ansible_engine_payload_7iohzesx/ansible_engine_payload.zip/ansible/module_utils/stonesoft_util.py\", line 504, in __init__\n  File \"/tmp/ansible_engine_payload_7iohzesx/__main__.py\", line 1169, in exec_module\n  File \"/home/seb/tmp/temp/env/lib/python3.7/site-packages/smc/core/engine.py\", line 763, in vpn_endpoint\n    return self.vpn.internal_endpoint\n  File \"/home/seb/tmp/temp/env/lib/python3.7/site-packages/smc/base/decorators.py\", line 59, in get\n    ret = self._cache[func] = func(self)\n  File \"/home/seb/tmp/temp/env/lib/python3.7/site-packages/smc/core/engine.py\", line 749, in vpn\n    return VPN(self)\n  File \"/home/seb/tmp/temp/env/lib/python3.7/site-packages/smc/core/engine.py\", line 1216, in __init__\n    self.internal_gateway = InternalGateway(**result[0])\nIndexError: list index out of range\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}
```

I'm using `develop` branch for both `smc-python` and `ansible-stonesoft`.

SMC version is 6.5.4.

IMHO, problem is in `smc-python`, I'll open an issue with informations I have.

**Comment by gabstopper** [comment]
Hmm. This is strange, I cannot duplicate the issue with smc-python. Will post my findings there!

**Comment by sebbbastien** [comment]
Problem solved in SMC 6.5.6.

See gabstopper/smc-python#52 for more details.
